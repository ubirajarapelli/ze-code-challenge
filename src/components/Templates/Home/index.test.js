import React from 'react';
import { render } from '@testing-library/react'
import { HomeTemplate } from './'

const renderComponent = () => {
  return render(
    <HomeTemplate>
      <h1>Home Template</h1>
    </HomeTemplate>
  )
}

describe('Test from HomeTemplate Component', () => {
  it('should make a snapshot from HomeTemplate Component', () => {
    const { container } = renderComponent()
    expect(container).toMatchSnapshot()
  })
})
