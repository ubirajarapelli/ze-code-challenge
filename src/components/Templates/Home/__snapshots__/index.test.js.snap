// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test from HomeTemplate Component should make a snapshot from HomeTemplate Component 1`] = `
<div>
  <header
    class="sc-dIsUp dPZYiy"
  >
    <nav
      class="sc-bqGGPW gvesvb"
    >
      <img
        alt="Logotipo Brewlivery"
        class="sc-ksluID jPHFdz"
      />
      <span />
      <button
        class="sc-hBMUJo sc-fFSPTT lodoa-d cvdwmV"
      >
        Login
      </button>
      <button
        aria-label="Ir para a sacola de compras"
        class="sc-hBMUJo sc-bkbkJK lodoa-d gpByXJ"
      >
        <svg
          fill="none"
          height="1em"
          stroke="currentColor"
          stroke-width="0"
          viewBox="0 0 24 24"
          width="1em"
          xmlns="http://www.w3.org/2000/svg"
        >
          <path
            clip-rule="evenodd"
            d="M5 4H19C19.5523 4 20 4.44771 20 5V19C20 19.5523 19.5523 20 19 20H5C4.44772 20 4 19.5523 4 19V5C4 4.44772 4.44771 4 5 4ZM2 5C2 3.34315 3.34315 2 5 2H19C20.6569 2 22 3.34315 22 5V19C22 20.6569 20.6569 22 19 22H5C3.34315 22 2 20.6569 2 19V5ZM12 12C9.23858 12 7 9.31371 7 6H9C9 8.56606 10.6691 10 12 10C13.3309 10 15 8.56606 15 6H17C17 9.31371 14.7614 12 12 12Z"
            fill="currentColor"
            fill-rule="evenodd"
          />
        </svg>
      </button>
    </nav>
  </header>
  <div
    class="sc-iJCRrE gAmRDA"
  >
    <main>
      <h1>
        Home Template
      </h1>
    </main>
    <section>
      <div
        class="sc-dIvrsQ eAkMvO"
      >
        <figure>
          <img
            alt="Cerveja Tupiniquin"
          />
          <figcaption
            class="sc-hHEiqL QMegq"
          >
            Microcervejarias especiais
            <br />
             com até 50% OFF
          </figcaption>
        </figure>
      </div>
    </section>
    <aside>
      <ul
        class="sc-cxNHIi eAwahD"
      >
        <li>
          <a
            aria-label="Link para a Brewlivery no Facebook"
            href="/facebook.com/brewlivery"
          >
            <svg
              fill="currentColor"
              height="1em"
              stroke="currentColor"
              stroke-width="0"
              viewBox="0 0 320 512"
              width="1em"
              xmlns="http://www.w3.org/2000/svg"
            >
              <path
                d="M279.14 288l14.22-92.66h-88.91v-60.13c0-25.35 12.42-50.06 52.24-50.06h40.42V6.26S260.43 0 225.36 0c-73.22 0-121.08 44.38-121.08 124.72v70.62H22.89V288h81.39v224h100.17V288z"
              />
            </svg>
          </a>
        </li>
        <li>
          <a
            aria-label="Link para a Brewlivery no Instagam"
            href="/instagam.com/brewlivery"
          >
            <svg
              fill="currentColor"
              height="1em"
              stroke="currentColor"
              stroke-width="0"
              viewBox="0 0 448 512"
              width="1em"
              xmlns="http://www.w3.org/2000/svg"
            >
              <path
                d="M224.1 141c-63.6 0-114.9 51.3-114.9 114.9s51.3 114.9 114.9 114.9S339 319.5 339 255.9 287.7 141 224.1 141zm0 189.6c-41.1 0-74.7-33.5-74.7-74.7s33.5-74.7 74.7-74.7 74.7 33.5 74.7 74.7-33.6 74.7-74.7 74.7zm146.4-194.3c0 14.9-12 26.8-26.8 26.8-14.9 0-26.8-12-26.8-26.8s12-26.8 26.8-26.8 26.8 12 26.8 26.8zm76.1 27.2c-1.7-35.9-9.9-67.7-36.2-93.9-26.2-26.2-58-34.4-93.9-36.2-37-2.1-147.9-2.1-184.9 0-35.8 1.7-67.6 9.9-93.9 36.1s-34.4 58-36.2 93.9c-2.1 37-2.1 147.9 0 184.9 1.7 35.9 9.9 67.7 36.2 93.9s58 34.4 93.9 36.2c37 2.1 147.9 2.1 184.9 0 35.9-1.7 67.7-9.9 93.9-36.2 26.2-26.2 34.4-58 36.2-93.9 2.1-37 2.1-147.8 0-184.8zM398.8 388c-7.8 19.6-22.9 34.7-42.6 42.6-29.5 11.7-99.5 9-132.1 9s-102.7 2.6-132.1-9c-19.6-7.8-34.7-22.9-42.6-42.6-11.7-29.5-9-99.5-9-132.1s-2.6-102.7 9-132.1c7.8-19.6 22.9-34.7 42.6-42.6 29.5-11.7 99.5-9 132.1-9s102.7-2.6 132.1 9c19.6 7.8 34.7 22.9 42.6 42.6 11.7 29.5 9 99.5 9 132.1s2.7 102.7-9 132.1z"
              />
            </svg>
          </a>
        </li>
        <li>
          <a
            aria-label="Link para a Brewlivery no Twitter"
            href="/twitter.com/brewlivery"
          >
            <svg
              fill="currentColor"
              height="1em"
              stroke="currentColor"
              stroke-width="0"
              viewBox="0 0 512 512"
              width="1em"
              xmlns="http://www.w3.org/2000/svg"
            >
              <path
                d="M459.37 151.716c.325 4.548.325 9.097.325 13.645 0 138.72-105.583 298.558-298.558 298.558-59.452 0-114.68-17.219-161.137-47.106 8.447.974 16.568 1.299 25.34 1.299 49.055 0 94.213-16.568 130.274-44.832-46.132-.975-84.792-31.188-98.112-72.772 6.498.974 12.995 1.624 19.818 1.624 9.421 0 18.843-1.3 27.614-3.573-48.081-9.747-84.143-51.98-84.143-102.985v-1.299c13.969 7.797 30.214 12.67 47.431 13.319-28.264-18.843-46.781-51.005-46.781-87.391 0-19.492 5.197-37.36 14.294-52.954 51.655 63.675 129.3 105.258 216.365 109.807-1.624-7.797-2.599-15.918-2.599-24.04 0-57.828 46.782-104.934 104.934-104.934 30.213 0 57.502 12.67 76.67 33.137 23.715-4.548 46.456-13.32 66.599-25.34-7.798 24.366-24.366 44.833-46.132 57.827 21.117-2.273 41.584-8.122 60.426-16.243-14.292 20.791-32.161 39.308-52.628 54.253z"
              />
            </svg>
          </a>
        </li>
      </ul>
    </aside>
  </div>
  <footer
    class="sc-crzoAE jHuWbY"
  >
    <div
      class="sc-iqAclL jjNvRh"
    >
      <p>
        Termos de uso e política de privacidade
      </p>
      <p>
        © 2021 Brewlivery.com
      </p>
      <p>
        Beba com moderção
      </p>
    </div>
  </footer>
</div>
`;
